src/Api/Polls.elm

```diff
@@ -188,15 +189,29 @@ update credentials poll newTitle transform =
         |> Task.map transform
 
 
-pollDecoder : Decoder Poll
+genericPollEndpoint : Credentials -> Api.Endpoint
+genericPollEndpoint credentials =
+    Api.authenticated credentials
+        |> Api.withPath "mod/"
+        |> Api.withPath (String.fromInt (Api.moderatorId credentials))
+        |> Api.withPath "/poll"
+
+
+specificPollEndpoint : PollDiscriminator -> (Credentials -> Api.Endpoint)
+specificPollEndpoint pollDiscriminator credentials =
+    genericPollEndpoint credentials
+        |> Api.withPath ("/" ++ String.fromInt pollDiscriminator.idPoll)
```

Comment:
I would maybe name these `anyPoll` and `somePoll`, that are shorter. Just a matter of taste tho 😄

Reply:
Why not, and with the strict types, no loss of clarity 😄